import-scripts/fetch-dmp-data-for-import.sh

```diff
@@ -195,43 +195,43 @@ printTimeStampedDataProcessingStepMessage "MSKIMPACT data processing"
 printTimeStampedDataProcessingStepMessage "Darwin CAISIS fetch for mskimpact"
 $JAVA_BINARY $JAVA_DARWIN_FETCHER_ARGS -s mskimpact -d $MSK_IMPACT_DATA_HOME -c
 if [ $? -gt 0 ] ; then
-    cd $MSK_IMPACT_DATA_HOME ; $HG_BINARY update -C ; find . -name "*.orig" -delete
+    bash $PORTAL_HOME/scripts/datasource-repo-cleanup.sh $PORTAL_DATA_HOME/dmp
     sendPreImportFailureMessageMskPipelineLogsSlack "MSKIMPACT Darwin CAISIS Fetch"
 else
     FETCH_DARWIN_CAISIS_FAIL=0
     echo "committing darwin caisis data"
-    cd $MSK_IMPACT_DATA_HOME ; $HG_BINARY commit -m "Latest MSKIMPACT Dataset: Darwin CAISIS"
+    cd $MSK_IMPACT_DATA_HOME ; $GIT_BINARY add ./* ; $GIT_BINARY commit -m "Latest MSKIMPACT Dataset: Darwin CAISIS"
 fi
 
 if [ $IMPORT_STATUS_IMPACT -eq 0 ] ; then
-    # fetch new/updated IMPACT samples using CVR Web service   (must come after mercurial fetching)
+    # fetch new/updated IMPACT samples using CVR Web service   (must come after git fetching)
     printTimeStampedDataProcessingStepMessage "CVR fetch for mskimpact"
     $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -d $MSK_IMPACT_DATA_HOME -n data_clinical_mskimpact_data_clinical_cvr.txt -i mskimpact -r 150 $CVR_TEST_MODE_ARGS
     if [ $? -gt 0 ] ; then
         echo "CVR fetch failed!"
-        cd $MSK_IMPACT_DATA_HOME ; $HG_BINARY update -C ; find . -name "*.orig" -delete
+        bash $PORTAL_HOME/scripts/datasource-repo-cleanup.sh $PORTAL_DATA_HOME/dmp
```

Comment:
this script wipes out built up commits, need a way to keep commits and just get rid of changes.